libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKViewUtils.m

```diff
@@ -28,6 +28,11 @@
 #import "SFSDKViewUtils.h"
 #import "SFSDKViewControllerConfig.h"
 
+#import "SFLoginViewController.h"
```

Comment:
Do we want to introduce dependency on internal view controllers for a utility class here? Is there a better way we could do this? What do you think, @bbirman and @wmathurin?

Reply:
An idea - not sure how it will work though, but can make an attempt.

This helper function can be passed a new param, a `Class` representing the class intended to be "controlled".

That would break the dependency (now, more generic) and it is possible `[UINavigationBar appearanceWhenContainedInInstancesOfClasses...` will _add_ the class rather than replacing others.

Reply:
I like that idea

Reply:
I may have some cycles this afternoon to look more closely at that, to see if it works.